frontend/components/main.js

```diff
@@ -121,16 +125,26 @@ export default class Main extends React.Component {
     } else if (isQueryTooShort) {
       return <div>Search term must have 2 or more characters</div>
     } else if (searchResults) {
-      return Object.entries(searchResults).map(([categoryId, definitions]) =>
-        this.renderCategory(findCategory(categoryId), definitions)
+      return (
+        <div>
+          <CategoryNav categories={categories} />
```

Comment:
In this condition, since the headings are there in the results, and the search term can be cleared, those seem like adequate ways back to browsing. It's a little confusing because the headings seem like they're part of the search results. What do you think about removing the nav menu here?

<img width="801" alt="screen shot 2019-01-10 at 8 53 30 pm" src="https://user-images.githubusercontent.com/1487036/51008485-f5941880-1519-11e9-84ec-fa0e41bcefb6.png">

<img width="862" alt="screen shot 2019-01-10 at 8 53 25 pm" src="https://user-images.githubusercontent.com/1487036/51008465-e3b27580-1519-11e9-8369-e2b131a14c65.png">